src/locales/ja/modifier-type.ts

```diff
@@ -0,0 +1,456 @@
+
+    "LOCK_CAPSULE": { name: "ロックカプセル", description: "リロールするときにアイテムのレアリティをロックできる" },
+
+    "GRIP_CLAW": { name: "ねばりのかぎづめ" },
+    "WIDE_LENS": { name: "こうかくレンズ" },
+
+    "MULTI_LENS": { name: "マルチレンズ" },
+
+    "HEALING_CHARM": { name: "ヒーリングチャーム", description: "HPをかいふくするわざとアイテムのこうかを10パーセントあげる (ふっかつはのぞく)" },
+    "CANDY_JAR": { name: "アメボトル", description: "ふしぎなアメのアイテムでふえるレベルが1ふえる" },
+
+    "BERRY_POUCH": { name: "きのみぶくろ", description: "つかったきのみがつかわれないかくりつを30パーセントふやす" },
+
+    "FOCUS_BAND": { name: "きあいのハチマキ", description: "ひんしになるダメージをうけてもHP1でたえるかくりつを10パーセントふやす" },
+
+    "QUICK_CLAW": { name: "せんせいのツメ", description: "すばやさにかかわらず さきにこうどうするかくりつを10パーセントふやす (ゆうせんどのあと)" },
+
+    "KINGS_ROCK": { name: "おうじゃのしるし", description: "こうげきわざがあいてをひるませるかくりつを10パーセントふやす" },
+
+    "LEFTOVERS": { name: "たべのこし", description: "ポケモンのさいだいHPの1/16をまいターンかいふくする" },
+    "SHELL_BELL": { name: "かいがらのすず", description: "ポケモンがあたえたダメージの1/8をかいふくする" },
+
+    "TOXIC_ORB": { name: "どくどくだま", description: "ターンの終わりに すでに じょうたいじょうしょうが なければ もうどくの じょうたいに なる" },
+    "FLAME_ORB": { name: "かえんだま", description: "ターンの終わりに すでに じょうたいじょうしょうが なければ やけどの じょうたいに なる" },
+
+    "BATON": { name: "バトン", description: "ポケモンをこうたいするときにこうかをひきつぎ わなをかいひすることもできる" },
+
+    "SHINY_CHARM": { name: "ひかるおまもり", description: "やせいのポケモンがいろちがいポケモンであるかくりつをおおきくふやす" },
+    "ABILITY_CHARM": { name: "とくせいおまもり", description: "やせいのポケモンがかくれとくせいをもつかくりつをおおきくふやす" },
+
+    "IV_SCANNER": { name: "こたいち たんちき", description: "やせいのポケモンのこたいちをスキャンできる。スタックごとに2つのこたいちがあきらかになる。もっともたかいこたいちがさいしょにひょうじされる" },
+
+    "DNA_SPLICERS": { name: "いでんしのくさび" },
+
+    "MINI_BLACK_HOLE": { name: "ミニブラックホール" },
+
+    "GOLDEN_POKEBALL": { name: "ゴールドモンスターボール", description: "バトルごとに1つのアイテムオプションをふやす" },
+
+    "ENEMY_DAMAGE_BOOSTER": { name: "ダメージトークン", description: "ダメージを5%ふやす" },
+    "ENEMY_DAMAGE_REDUCTION": { name: "プロテクショントークン", description: "うけるダメージを2.5%へらす" },
+    "ENEMY_HEAL": { name: "かいふくトークン", description: "まいターンさいだいHPの2%をかいふくする" },
+    "ENEMY_ATTACK_POISON_CHANCE": { name: "どくトークン" },
+    "ENEMY_ATTACK_PARALYZE_CHANCE": { name: "まひトークン" },
+    "ENEMY_ATTACK_BURN_CHANCE": { name: "やけどトークン" },
+    "ENEMY_STATUS_EFFECT_HEAL_CHANCE": { name: "なおしトークン", description: "まいターン2.5%のかくりつでじょうたいじょうしょうをかいふくする" },
+    "ENEMY_ENDURE_CHANCE": { name: "こらえるトークン" },
+    "ENEMY_FUSED_CHANCE": { name: "フュージョントークン", description: "やせいのポケモンがフュージョンするかくりつを1%ふやす" },
+  },
+  SpeciesBoosterItem: {
+    "LIGHT_BALL": { name: "Light Ball", description: "It's a mysterious orb that boosts Pikachu's Attack and Sp. Atk stats." },
+    "THICK_CLUB": { name: "Thick Club", description: "This hard bone of unknown origin boosts Cubone or Marowak's Attack stat." },
+    "METAL_POWDER": { name: "Metal Powder", description: "Extremely fine yet hard, this odd powder boosts Ditto's Defense stat." },
+    "QUICK_POWDER": { name: "Quick Powder", description: "Extremely fine yet hard, this odd powder boosts Ditto's Speed stat." }
```

Comment:
```suggestion
    "LIGHT_BALL": { name: "でんきだま", description: "ピカチュウに 持たせると 攻撃と 特攻が あがる 不思議な玉。" },
    "THICK_CLUB": { name: "ふといホネ", description: "なにかの 硬いホネ。カラカラ または ガラガラに 持たせると 攻撃が あがる。" },
    "METAL_POWDER": { name: "メタルパウダー", description: "メタモンに 持たせると 防御が あがる 不思議な粉。とても こまかくて 硬い。" },
    "QUICK_POWDER": { name: "スピードパウダー", description: "メタモンに 持たせると 素早さが あがる 不思議 粉。とても こまかくて 硬い。" }
```
Localization for the `SpeciesBoosterItem`s using the same source from their initial PR (https://github.com/pagefaultgames/pokerogue/pull/1899#issuecomment-2168838329).